using-mobile-extensions/adobe-edge-identity/edge-identity-faq.md

```diff
@@ -125,4 +125,184 @@ func application(_ application: UIApplication, didFinishLaunchingWithOptions lau
 }
 ```
 {% endtab %}
-{% endtabs %}
\ No newline at end of file
+{% endtabs %}
+
+
+## Q: Will an existing Experience Cloud ID (ECID) migrate to the AEP Edge Identity extension?
+### A: Yes
+If the application previously installed the AEP Identity direct solution extension and upgrades to the AEP Edge Identity extension, the existing ECID value is migrated to the AEP Edge Identity extension on first launch of the application.
+
+Note, however, if the Mobile SDK's privacy status was set to `optedOut` at the time the application is upgraded, the AEP Identity direct solution extension will not have an ECID, as it was cleared. In this case, the AEP Edge Identity extension will generate a new ECID.
+
+
+## Q: What is the Experience Cloud ID (ECID) used by the SDK when using both AEP Edge extensions and Adobe Solutions extensions?
+### A: The AEP Edge Identity extension and the AEP Identity direct solution extension each manage their own ECID.
```

Comment:
their own ECID. However, the two ECIDs are synced as part of the XDM IdentityMap.